stream-chat-android/src/main/res/layout/stream_dialog_message_moreaction.xml

```diff
@@ -188,13 +197,14 @@
                 />
 
             <TextView
-                style="@style/text"
+                android:id="@+id/deleteMessageButtonTextView"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:gravity="center_vertical"
                 android:text="@string/stream_more_action_delete"
-                android:textColor="#8A000000"
+                android:textColor="@color/stream_black_54"
+                android:textSize="15sp"
```

Comment:
Could we declare a common @style for those `TextView`s including `gravity`, `textColor`, and `textSize` ?